spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature 'comments' do
+
+  scenario 'user must be logged in to create a comment' do
+    user = build(:user)
+    sign_up(user)
+    make_post 'whatever bro'
+    click_link 'Sign out'
+    visit '/posts'
+    click_link 'Comment'
+    expect(page).not_to have_content 'Leave Comment'
+    expect(current_path).to eq '/users/sign_in'
+  end
+
+  scenario 'allows users to leave a comment' do
+    user = build(:user)
+    sign_up(user)
+    make_post 'hey'
+    make_comment 'hey yourself'
+    expect(current_path).to eq '/posts'
+    expect(page).to have_content('hey yourself')
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)